mvc/tutorials/mvc-with-entity-framework.rst

```diff
@@ -27,7 +27,7 @@ In this article:
     - `Add a delete page`_
     - `Wrapping up`_
 
-You can browse the source code for the sample app on `GitHub <https://github.com/aspnet/Docs/tree/master/docs/mvc/tutorials/mvc-with-entity-framework/sample>`_.
+You can browse the source code for the sample app on `GitHub <https://github.com/aspnet/Docs/tree/master/mvc/tutorials/mvc-with-entity-framework/sample>`__.
```

Comment:
http://stackoverflow.com/a/14067756/1261807

I guess the other option is a ref but this seemed simpler.